feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```

Comment:
Its good you have decided to add this line

Reply:
Thanks for the comment